docs/sources/tempo/configuration/_index.md

```diff
@@ -361,8 +367,8 @@ metrics_generator:
             [- <Prometheus remote write config>]
 
     # This option only allows spans with start time that occur within the configured duration to be
```

Comment:
this was my mistake but if you can, can you change this to "end time" instead of "start time"

Reply:
Will do.